packages/terra-application/src/terra-dev-site/test/terra-application/application-container/SkipToLinks.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
-import ApplicationBase from '../../../application-base';
-import ApplicationContainer from '../../../application-container';
-import SkipToLink from '../../../application-container/private/skip-to-links/SkipToLink';
+import ApplicationBase from '../../../../application-base';
```

Comment:
the terra-application tests have been scoped under a terra-application folder now.